Subject: [PATCH] imguicfg switch patch
---
Index: imconfig.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/imconfig.h b/imconfig.h
--- a/imconfig.h	(revision 947aa9c9722bd6ff740dd757da609ff41f4d3ba3)
+++ b/imconfig.h	(date 1740580107574)
@@ -140,3 +140,15 @@
     void MyFunction(const char* name, MyMatrix44* mtx);
 }
 */
+#ifdef __SWITCH__
+#define IMGUI_IMPL_OPENGL_LOADER_CUSTOM
+#define GL_GLEXT_PROTOTYPES 1
+#include <glad/glad.h>
+#endif

