Subject: [PATCH] SWITCH-IMGUI
---
Index: imgui.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/imgui.cpp b/imgui.cpp
--- a/imgui.cpp	(revision 947aa9c9722bd6ff740dd757da609ff41f4d3ba3)
+++ b/imgui.cpp	(date 1740681739331)
@@ -20612,6 +20612,10 @@
 #define IMGUI_DISABLE_DEFAULT_SHELL_FUNCTIONS
 #endif
 
+#if defined(__SWITCH__)
+#define IMGUI_DISABLE_DEFAULT_SHELL_FUNCTIONS
+#endif
+
 #if defined(_WIN32) && defined(IMGUI_DISABLE_WIN32_FUNCTIONS)
 #define IMGUI_DISABLE_DEFAULT_SHELL_FUNCTIONS
 #endif
@@ -20632,6 +20636,7 @@
 #include <unistd.h>
 static bool Platform_OpenInShellFn_DefaultImpl(ImGuiContext*, const char* path)
 {
+
 #if defined(__APPLE__)
     const char* args[] { "open", "--", path, NULL };
 #else
